applications/plugins/picopass/helpers/iclass_elite_dict.c

```diff
@@ -0,0 +1,151 @@
+#include "iclass_elite_dict.h"
+
+#include <lib/toolbox/args.h>
+#include <lib/flipper_format/flipper_format.h>
+
+#define ICLASS_ELITE_DICT_FLIPPER_PATH EXT_PATH("picopass/assets/iclass_elite_dict.txt")
```

Comment:
I wonder if we want to make this ANY_PATH and include an empty file, or maybe the `5b7c62c491c11b39` value from proxmark?

Reply:
I think there is some refactoring that could happen and add other keys from pm3. It would be nice to detect unpersonalized cards as well.